lib/Runtime/Library/JavascriptErrorDebug.cpp

```diff
@@ -235,10 +239,7 @@ namespace Js
         memset(proerrstr, 0, sizeof(*proerrstr));
         *pperrinfo = nullptr;
 
-#ifndef _WIN32
-        // xplat-todo: Find out if we can implement richer error info on Linux
-        return hr;
-#else
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
AFAIK '_MSC_VER' defined means '__clang__' is not defined. What option we are trying to rule out here?